commit bb72da0bc9c38d32656c0cbf636b68c39acf52bf (refs/changes/22/4622/2)
Author: Pedro Palazon Candel <pedro@joyent.com>
Date:   2018-08-06T17:38:43+02:00 (1 year, 2 months ago)
    
    TOOLS-1964 sdc's zone SAPI template should use RabbitMQ domain instead of hardcoded IP
    Reviewed by: Marsell Kukuljevic <marsell@joyent.com>
    Approved by: Marsell Kukuljevic <marsell@joyent.com>

diff --git a/sapi_manifests/sdc/template b/sapi_manifests/sdc/template
index 9f4684f..c00213b 100644
--- a/sapi_manifests/sdc/template
+++ b/sapi_manifests/sdc/template
@@ -11,7 +11,7 @@
     "mahi_domain": "mahi.{{{datacenter_name}}}.{{{dns_domain}}}",
     "napi_domain": "{{{napi_domain}}}",
     "papi_domain": "{{{papi_domain}}}",
-    "rabbitmq": "{{{rabbitmq}}}",
+    "rabbitmq": "guest:guest:{{{rabbitmq_domain}}}:5672",
     "sapi_domain": "{{{sapi_domain}}}",
     "sdc_key_id": "{{{SDC_KEY_ID}}}",
     "ufds_admin_login": "{{{ufds_admin_login}}}",
